sites/footers/reports/a11y-1.json

```diff
@@ -0,0 +1,905 @@
+        "WCAG21Tech:general/G166",
+        "WCAG21Tech:failures/F30",
+        "WCAG21Tech:failures/F67"
+      ],
+      "earl:mode": "earl:automatic",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 1.2.2 Captions (Prerecorded) (Level A)",
+      "earl:test": "WCAG20:captions-prerecorded",
+      "outcome": "Pass",
+      "earl:outcome": "earl:passed",
+      "notes": "",
+      "techniques": [
+        "WCAG21Tech:general/G93",
+        "WCAG21Tech:general/G87",
+        "WCAG21Tech:failures/F8",
+        "WCAG21Tech:failures/F75",
+        "WCAG21Tech:failures/F74"
+      ],
+      "earl:mode": "earl:automatic",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 1.2.3 Audio Description or Media Alternative (Prerecorded) (Level A)",
+      "earl:test": "WCAG20:audio-description-or-media-alternative-prerecorded",
+      "outcome": "Pass",
+      "earl:outcome": "earl:passed",
+      "notes": "",
+      "techniques": [
+        "WCAG21Tech:general/G69",
+        "WCAG21Tech:general/G68",
+        "WCAG21Tech:general/G78",
+        "WCAG21Tech:general/G173",
+        "WCAG21Tech:general/G8",
+        "WCAG21Tech:general/G203"
+      ],
+      "earl:mode": "earl:automatic",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 1.2.4 Captions (Live) (Level AA)",
+      "earl:test": "WCAG20:captions-live",
+      "outcome": "Pass",
+      "earl:outcome": "earl:passed",
+      "notes": "",
+      "techniques": [
+        "WCAG21Tech:general/G9",
+        "WCAG21Tech:general/G93",
+        "WCAG21Tech:general/G87"
+      ],
+      "earl:mode": "earl:automatic",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 1.2.5 Audio Description (Prerecorded) (Level AA)",
+      "earl:test": "WCAG20:captions-live",
+      "outcome": "Pass",
+      "earl:outcome": "earl:passed",
+      "notes": "",
+      "techniques": [
+        "WCAG21Tech:general/G78",
+        "WCAG21Tech:general/G173",
+        "WCAG21Tech:general/G8",
+        "WCAG21Tech:general/G203"
+      ],
+      "earl:mode": "earl:automatic",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 1.3.1 Info and Relationships (Level A)",
+      "earl:test": "WCAG20:info-and-relationships",
+      "outcome": "Pass",
+      "earl:outcome": "earl:passed",
+      "notes": "",
+      "techniques": [
+        "WCAG21Tech:general/G115",
+        "WCAG21Tech:general/G117",
+        "WCAG21Tech:general/G140",
+        "WCAG21Tech:failures/F2",
+        "WCAG21Tech:failures/F33",
+        "WCAG21Tech:failures/F34",
+        "WCAG21Tech:failures/F42",
+        "WCAG21Tech:failures/F43",
+        "WCAG21Tech:failures/F46",
+        "WCAG21Tech:failures/F48",
+        "WCAG21Tech:failures/F87",
+        "WCAG21Tech:failures/F90",
+        "WCAG21Tech:failures/F91",
+        "WCAG21Tech:failures/F92"
```

Comment:
According to the recently updated instruction for producing the JSON-LD report, we should only include the technique that is being discussed in the evaluation note.
Here considering this SC is passing, there should be no failure technique listed.

```suggestion
```

Reply:
Those technique outcome was marked "can't tell"